code/datums/abilities/lizard.dm

```diff
@@ -75,6 +75,7 @@
 /datum/targetable/lizardAbility/regrow_tail
 	name = "Regrow Tail"
 	desc = "Regrow your tail... (If cast while you have a tail, shoot off your tail and regrow a new one)"
+	cooldown = 3000
```

Comment:
use the time defines

Reply:
like it's literally easier to do them then figure out the math for this value, smh

Reply:
Donezo.